applications/sim_showers_for_trigger_rates.py

```diff
@@ -0,0 +1,186 @@
+
+    Example
+    -------
+    Producing a set of proton showers for trigger rate simulations of LST.
+
+    .. code-block:: console
+
+        python applications/sim_showers_for_trigger_rates.py -a 4LST -s North \
+        --primary proton --nruns 100 --nevents 10000 --output {some dir for large files}
+'''
+
+import logging
+import argparse
+
+import astropy.units as u
+
+import simtools.io_handler as io
+import simtools.config as cfg
+import simtools.util.general as gen
+from simtools.shower_simulator import ShowerSimulator
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description=(
+            'Calculate and plot the PSF and eff. mirror area as a function of off-axis angle '
+            'of the telescope requested.'
+        )
+    )
+    parser.add_argument(
+        '-a',
+        '--array',
+        help='Name of the array (e.g. 1MST, 4 LST ...)',
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        '-s',
+        '--site',
+        help='Site name (North or South)',
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        '--primary',
+        help='Name of the primary particle (e.g. proton, helium ...)',
+        type=str,
+        required=True
+    )
+    parser.add_argument(
+        '--nruns',
+        help='Number of runs (default=100)',
+        type=int,
+        default=100
+    )
+    parser.add_argument(
+        '--nevents',
+        help='Number of events/run (default=100)',
+        type=int,
+        default=100000
+    )
+    parser.add_argument(
+        '--zenith',
+        help='Zenith angle in deg (default=20)',
+        type=float,
+        default=20
+    )
+    parser.add_argument(
+        '--azimuth',
+        help='Azimuth angle in deg (default=0)',
+        type=float,
+        default=0
+    )
+    parser.add_argument(
+        '--output',
+        help='Path of the output directory where the simulations will be saved.',
+        type=str,
+        default=None
+    )
+    parser.add_argument(
+        '--test',
+        help='Test option will not submit any job.',
+        action='store_true'
+    )
+    parser.add_argument(
+        '-v',
+        '--verbosity',
+        dest='logLevel',
+        action='store',
+        default='info',
+        help='Log level to print (default is INFO)'
+    )
+
+    args = parser.parse_args()
+    label = 'trigger_rates'
+
+    logger = logging.getLogger()
+    logger.setLevel(gen.getLogLevelFromUser(args.logLevel))
+
+    # Output directory to save files related directly to this app
+    outputDir = io.getApplicationOutputDirectory(cfg.get('outputLocation'), label)
+
+    showerConfigData = {
+        'corsikaDataDirectory': args.output,
+        'site': args.site,
+        'layoutName': args.array,
+        'runRange': [1, args.nruns + 1],
+        'nshow': args.nevents,
+        'primary': args.primary,
+        'erange': [100 * u.GeV, 300 * u.TeV],
+        'eslope': -2,
```

Comment:
(I think I've missed where the parameters are set; these are simply default values?)

-2 is obviously not a good slope for trigger estimation, or is there a reweighting in sim_telarray?

Reply:
These are default of the application. These are parameters that are not supposed to change when running this specific application.

Yes, there is a reweighting in sim_telarray and this can also be done when calculating the trigger rates.

Reply:
(see question earlier, so I resolve this one)